pkg/config/config.go

```diff
@@ -586,6 +589,34 @@ func InitializeTestingConfig() {
 	SetDebugMode(true)
 	SetDataPath("data/")
 }
+func ReadRunModConfig() bool {
```

Comment:
don't return bool here, but rather return the error itself. We shouldn't "convert" the error that are returned by the underlying functions and convert it to a "bool